build/lint.mk

```diff
@@ -0,0 +1,73 @@
+#
+# Makefile fragment for Testing
```

Comment:
Not a blocker by any means, but did you want this to read "Makefile fragment for Linting"? Looks like a copy paste thing from the other fragment file.